bpf/mock/Makefile

```diff
@@ -0,0 +1,59 @@
+# Copyright 2017-2020 Authors of Cilium
+# SPDX-License-Identifier: Apache-2.0
+
+CMOCK ?= cmock.rb
```

Comment:
It'd be helpful to clarify what this file is, I don't see in this PR and from a quick search it appears to be part of CMake. At first I wasn't sure, and wondered if it was a new invention :)